packages/hash/api/src/graphql/resolvers/knowledge/model-mapping.ts

```diff
@@ -43,6 +42,7 @@ export const mapPageModelToGQL = (
   pageModel: PageModel,
 ): UnresolvedPersistedPageGQL => ({
   ...mapEntityModelToGQL(pageModel),
+  entityType: mapEntityTypeModelToGQL(pageModel.entityTypeModel),
```

Comment:
Doesn't block merging, but I would prefer we get rid of this field entirely if we're now removing it from `PersistedEntity`. It shouldn't result in significant additions to the diff as the `entityType` field should not currently be queried on the `PersistedPage`/`PersistedBlock` types.

```suggestion
```

Reply:
I'd prefer to not touch these typedefs considering we have an asana task dedicated to reworking them. https://app.asana.com/0/0/1203214689883093/f

While we're changing _some_ of their code, it's limited to one side and not a breaking change.

Reply:
Okay, still not really sure why these changes are happening in different PRs but don't feel strongly so let's do as you prefer.